library/system/src/lib/yast2/system_service.rb

```diff
@@ -76,7 +76,7 @@ def socket
       socket_name = socket_name[/\S+\.socket/]
       return unless socket_name # triggered by non-socket
 
-      @socket = Yast::SystemdSocket.find(socket_name)
+      @socket = Yast::Systemd::Socket.find(socket_name)
```

Comment:
Does it work? It should be `@socket = Systemd::Socket.find(socket_name`

Reply:
Oops :hankey:! Good catch!